indigo_app/static/javascript/indigo/views/work.js

```diff
@@ -49,6 +49,8 @@
       '#id_work-publication_name': 'publication_name',
       '#id_work-publication_number': 'publication_number',
       '#id_work-stub': 'stub',
+      // Including the below leads to 'Commenced' being unchecked when the form loads; no idea why.
+      '#id_work-commenced': 'commenced',
```

Comment:
This is working for me, perhaps it was earlier on when the `commenced` field wasn't being serialised yet.